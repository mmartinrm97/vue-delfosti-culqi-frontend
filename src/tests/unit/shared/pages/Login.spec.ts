import { describe, it, expect } from 'vitest'
import { mount } from '@vue/test-utils'
import LoginViewVue from '@/pages/LoginView.vue'

describe('LoginPage page', () => {
  it('should render', () => {
    const wrapper = mount(LoginViewVue)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
