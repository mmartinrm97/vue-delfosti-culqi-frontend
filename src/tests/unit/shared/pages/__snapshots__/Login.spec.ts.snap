// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`LoginPage page > should render 1`] = `
"<!-- Onboarding Section -->
<div class=\\"flex w-1/2 flex-col\\">
  <!-- Image Section-->
  <div class=\\"relative flex h-2/3 items-center justify-center\\"><img alt=\\"Culqi Login Cover\\" class=\\"absolute h-full w-full object-cover\\" src=\\"/src/assets/images/culqi-login-cover.png\\"></div><!--      Separator-->
  <!--            &lt;div class=&quot;flex items-center justify-between&quot;&gt;-->
  <!--              &lt;hr class=&quot;w-full border-4 border-primary&quot; /&gt;-->
  <!--            &lt;/div&gt;-->
  <!--          Text Section-->
  <div class=\\"items-left flex h-1/3 flex-col justify-center border-t-8 border-primary bg-secondary px-20\\">
    <div class=\\"h-auto w-24 pb-4\\">
      <!-- &lt;img alt=&quot;Culqi Logo&quot; src=&quot;@/assets/images/culqi_logo_white.svg&quot; /&gt; -->
    </div>
    <div class=\\"text-left\\"><span class=\\"text-3xl font-bold text-gray-50\\"> Dale mÃ¡s power âš¡ a tus empleados hoy ðŸ’ª </span>
      <p class=\\"py-4 font-light leading-7 text-gray-50\\"> Te ayudamos a gestionarlos de manera mÃ¡s sencilla </p>
    </div>
  </div>
</div>
<!-- Sign In Section -->
<div class=\\"flex w-1/2 flex-col justify-center\\">
  <div class=\\"my-auto flex flex-col justify-center px-8 lg:px-14 xl:px-24\\">
    <!--        Content Section-->
    <div class=\\"pt-40\\">
      <h1 class=\\"whitespace-nowrap text-center text-3xl font-bold leading-loose tracking-wide\\"> Inicia sesiÃ³n </h1>
      <form>
        <div class=\\"pt-6\\"><label class=\\"font-medium\\" for=\\"email\\">Correo ElectrÃ³nico <span class=\\"text-red-500\\">*</span></label>
          <div class=\\"mt-2 flex w-full items-center overflow-hidden rounded-xl border border-gray-300 transition-all focus-within:border-success\\"><input id=\\"email\\" autofocus=\\"\\" class=\\"w-full border-0 p-4 font-light focus:outline-none focus:ring-0\\" name=\\"email\\" placeholder=\\"Ingresa el correo electrÃ³nico\\" type=\\"text\\" autocomplete=\\"email\\"></div>
        </div>
        <div class=\\"pt-6\\"><label class=\\"font-medium\\" for=\\"password\\">ContraseÃ±a<span class=\\"text-red-500\\">*</span></label>
          <div class=\\"mt-2 flex w-full items-center overflow-hidden rounded-xl border border-gray-300 transition-all focus-within:border-success\\"><input id=\\"password\\" class=\\"w-full border-0 p-4 font-light focus:outline-none focus:ring-0\\" name=\\"password\\" placeholder=\\"Ingresa la contraseÃ±a\\" type=\\"password\\" autocomplete=\\"current-password\\"></div>
          <!--v-if-->
        </div>
        <div class=\\"pt-8\\"><button class=\\"w-full flex items-center space-x-6 justify-center rounded-xl bg-secondary px-8 py-4 text-white shadow-lg hover:bg-secondary/80 focus:outline-none focus:ring-4 focus:ring-primary\\" type=\\"submit\\"><span class=\\"text-lg\\">Iniciar SesiÃ³n</span>
            <!--v-if-->
          </button></div>
      </form>
      <div class=\\"pt-8\\">
        <div class=\\"text-center font-light text-gray-500\\"> Â¿Eres nuevo aquÃ­? <a class=\\"font-semibold text-primary hover:text-primary/80\\" href=\\"#\\"> Crea una cuenta </a></div>
      </div>
    </div>
  </div><!-- Terms Section -->
  <div class=\\"flex flex-col items-center justify-center whitespace-nowrap p-10 text-center\\"><span class=\\"text-gray-500\\">Â© 2023 Culqi. Todos los derechos reservados.</span></div>
</div>"
`;
